Sources/RugbyFoundation/Core/Use/SupportFilesPatcher.swift

```diff
@@ -39,18 +39,19 @@ final class SupportFilesPatcher {
                            replacement: binaryFolderPath))
 
             guard let moduleName = product.moduleName else { return }
+            let name = product.name
             let moduleMap = "\(moduleName).modulemap"
             result.append(("\(PODS_CONFIGURATION_BUILD_DIR)\(parentFolderName)/\(moduleMap)",
                            replacement: "\(binaryFolderPath)/\(moduleMap)"))
 
             if product.type == .staticLibrary {
                 result.append((
-                    "\(PODS_XCFRAMEWORKS_BUILD_DIR)\(moduleName)/\(HEADERS)",
-                    replacement: "\(binaryFolderPath)/\(moduleName)/\(HEADERS)"
+                    "\(PODS_XCFRAMEWORKS_BUILD_DIR)\(name)/\(HEADERS)",
+                    replacement: "\(binaryFolderPath)/\(name)/\(HEADERS)"
```

Comment:
Let's use `product.name` directly in `"\(PODS_XCFRAMEWORKS_BUILD_DIR)\(product.name)/\(HEADERS)"`.
It looks like the `name` is redundant variable.

```suggestion
                    "\(PODS_XCFRAMEWORKS_BUILD_DIR)\(product.name)/\(HEADERS)",
                    replacement: "\(binaryFolderPath)/\(product.name)/\(HEADERS)"
```

Reply:
Fixed